static/app/views/dashboardsV2/manage/index.tsx

```diff
@@ -87,17 +98,30 @@ class ManageDashboards extends AsyncView<Props, State> {
 
   renderDashboards() {
     const {dashboards, dashboardsPageLinks} = this.state;
-    const {organization, location} = this.props;
+    const {organization, location, api} = this.props;
     return (
       <DashboardList
+        api={api}
         dashboards={dashboards}
         organization={organization}
         pageLinks={dashboardsPageLinks}
         location={location}
+        onDashboardsChange={() => this.onDashboardsChange()}
       />
     );
   }
 
+  onCreate() {
+    const {organization} = this.props;
+    browserHistory.push({
+      pathname: `/organizations/${organization.slug}/dashboards/`,
+      state: {
```

Comment:
Is passing state in location and relying on that to enter ["create" mode](https://github.com/getsentry/sentry/pull/25558/files#diff-af8c8438bd2c80266943452229b652c5b70c1ced47ee0135b5dd7bd53ffb8824R64) ok?

Reply:
It's one approach, but probably not ideal.